DQM/RPCMonitorDigi/src/SealModule.cc

```diff
@@ -7,5 +7,3 @@ DEFINE_FWK_MODULE(RPCMonitorDigi);
 DEFINE_FWK_MODULE(RPCRecHitProbability);
 #include "DQM/RPCMonitorDigi/interface/RPCTTUMonitor.h"
 DEFINE_FWK_MODULE(RPCTTUMonitor);
-#include "DQM/RPCMonitorDigi/interface/RPCDcsInfo.h"
-DEFINE_FWK_MODULE(RPCDcsInfo);
```

Comment:
If you are making it so the module can never be loaded, then shouldn't RPCDcsInfo.h and RPCDcsInfo.cc both be removed from CMSSW?

Reply:
I forgot to update the removal... Thank you for mentioning that!